tests/test.js

```diff
@@ -0,0 +1,267 @@
+var assert    = require('assert'),
```

Comment:
There's a mixture of string styles in this file and elsewhere - some use ' and other ". Our project standard is for " - hopefully the jshint task will pick this up.